pages/message-board.html

```diff
@@ -70,5 +106,8 @@ <h1>The Regis Altare Birthday Project 2023</h1>
         >
       </footer>
     </main>
+    <script type="text/javascript" src="../js/papaparse.min.js"></script>
+    <script type="text/javascript" src="../js/a11y-dialog.min.js"></script>
+    <script type="module" src="../js/main.js"></script>
```

Comment:
we probably won't need papaparse for the message board. Unless it needs to be included because main.js requires it?

Reply:
yeah that was the reason why I included papaparse. I could separate the nav js event handlers into a separate file to reduce the amount of stuff included for pages that don't need papaparse.

Reply:
If it wouldn't take much time, that sounds better!